logging/structured/src/main/scala/tofu/logging/Logs.scala

```diff
@@ -48,13 +78,24 @@ object Logs {
         Sync[I].delay(new ContextSyncLoggingImpl[F, ctx.Ctx](ctx.context, LoggerFactory.getLogger(name)))
     }
   }
+
+  /**
+    * Returns an instance of [[tofu.logging.Logs]] that will produce the same constant [[tofu.logging.Logging]] instances.
+    */
   def const[I[_]: Applicative, F[_]](logging: Logging[F]): Logs[I, F] = new Logs[I, F] {
     def forService[Svc: ClassTag]: I[Logging[F]] = logging.pure[I]
     def byName(name: String): I[Logging[F]]      = logging.pure[I]
   }
 
+  /**
+    * Retuns an instance of [[tofu.logging.Logs]] that will produce a no-op [[tofu.logging.Logging]] instances.
+    */
   def empty[I[_]: Applicative, F[_]: Applicative]: Logs[I, F] = const[I, F](Logging.empty[F])
 
+  /**
+    * Combines two instances of [[tofu.logging.Logs]], resulting in a single one that will produce
```

Comment:
> Combines to produce instances by combining. 

It looks useless to me. Imho it's better to tell about the behavior of the result.